app/views/ctc/ctc_pages/puerto_rico.html.erb

```diff
@@ -0,0 +1,136 @@
+<% content_for :page_title do %>
```

Comment:
It looks like a lot of this is the same of the regular home page. To make them easier maintain do you think it would make sense to extract some partials for the parts that are the same?